worklist/utility.py

```diff
@@ -0,0 +1,93 @@
+import math
+import logging
+from collections import defaultdict
+import requests
+import json
+
+logger = logging.getLogger('django')
+
+
+# Function to convert a comma separated
+# string into list of strings
+def convert_string_to_list(input_string):
```

Comment:
The purpose of writing them as separate functions was just to avoid writing that line every time. And they'll be used in the code for MVP.
Anyhow, I've removed them for now.